community_supplied/Kernel/kernel-tcpip/check-arp-cache-mgmt-count-iagent.rule

```diff
@@ -0,0 +1,181 @@
+            }
+            sensor arp-mgmt-cache-count-sensor {
+                synopsis "Sensor to check arp management cache count";
+                description "Sensor to check arp management cache count";
+                iAgent {
+                    file arp_mgmt_cache.yml;
+                    table checkArpMgmtCacheTable;
+                    frequency 1m;
+                }
+            }
+            field arp-cache-mgmt-critical-percent {
+                constant {
+                    value "{{arp-cache-mgmt-critical-percent-input}}";
+                }
+                description "arp management cache percentage critical limit";
+            }
+            field arp-cache-mgmt-warning-percent {
+                constant {
+                    value "{{arp-cache-mgmt-warning-percent-input}}";
+                }
+                description "arp management cache percentage warning limit";
+            }
+            field nbrcache-mgmt-cnt {
+                sensor arp-mgmt-cache-count-sensor {
+                    path arpcache_mgmt_cnt;
+                }
+                type integer;
+                description "Current arp management cache count in the system";
+            }
+            field nbrcache-mgmt-drop-cnt {
+                sensor arp-mgmt-cache-count-sensor {
+                    path arpcache_mgmt_drop_cnt;
+                }
+                type integer;
+                description "Maximum arp management cache drop count in the system";
+            }
+            field nbrcache-mgmt-max {
+                sensor arp-mgmt-cache-count-sensor {
+                    path arpcache_mgmt_max_cnt;
+                }
+                type integer;
+                description "Maximum arp management cache count in the system";
+            }
+            field output-critical {
+                formula {
+                    user-defined-function {
+                        function-name arp-cache-mgmt-status;
+                        argument max_val "$nbrcache-mgmt-max";
+                        argument actual_val "$nbrcache-mgmt-cnt";
+                        argument percent_val "$arp-cache-mgmt-critical-percent";
+                    }
+                }
+                type integer;
+            }
+            field output-warning {
+                formula {
+                    user-defined-function {
+                        function-name arp-cache-mgmt-status;
+                        argument max_val "$nbrcache-mgmt-max";
+                        argument actual_val "$nbrcache-mgmt-cnt";
+                        argument percent_val "$arp-cache-mgmt-warning-percent";
+                    }
+                }
+                type integer;
+            }
+            trigger report-arp-mgmt-cache-count-iagent {
+                synopsis "Trigger to check arp management cache count";
+                description "Trigger to check arp management cache count";
+                frequency 1m;
```

Comment:
Always ensure trigger frequency is atleast 1.5 times of sensor frequency. Modify trigger frequency